.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Question
+about: Have any questions about how Jekyll works?
+---
+
+<!--
+  The Jekyll issue tracker IS NOT for usage questions! Please post your
+  question on our dedicated forum at http://talk.jekyllrb.com.
```

Comment:
We can link to HTTPS here. https://talk.jekyllrb.com/

Reply:
i just remember our certificate running out every couple of months, so i was wary to link https

Reply:
I know you have been telling people to ask questions on Jekyll Talk...yet there's now a template explicitly for this purpose. In the chooser that appears, I assume this will either be hidden or just a pointer to the other site?

In any case, I can imagine scenarios where issues get opened because people run into a problem and are unsure what caused it. The Support tag has a bunch of these, where directions were either not followed or people are asking questions about what a given error means and how to get out of a particular hole.

In fairness, I've done this myself a few times 🙃

Reply:
@chrisfinazzo yeah, it's supposed to be a pointer to talk.jekyllrb.com. we can update the description as it appears in the picker to already point there, i guess?